src/lib/firebase.js

```diff
@@ -4,15 +4,16 @@ import firebase from 'firebase/app';
 import 'firebase/firestore';
 
 // Initalize Firebase.
-const firebaseConfig = {
-  apiKey: "AIzaSyDlgVTZkjq6HT0pFm7sBiSm60EQ19D2RrI",
-  authDomain: "tcl-26-shopping-list.firebaseapp.com",
-  projectId: "tcl-26-shopping-list",
-  storageBucket: "tcl-26-shopping-list.appspot.com",
-  messagingSenderId: "368940430821",
-  appId: "1:368940430821:web:5347ad9d833f3a84cd6b0c"
-};
+if (!firebase.apps.length) {
+  firebase.initializeApp({
+    apiKey: 'AIzaSyDlgVTZkjq6HT0pFm7sBiSm60EQ19D2RrI',
+    authDomain: 'tcl-26-shopping-list.firebaseapp.com',
+    projectId: 'tcl-26-shopping-list',
+    storageBucket: 'tcl-26-shopping-list.appspot.com',
+    messagingSenderId: '368940430821',
+    appId: '1:368940430821:web:5347ad9d833f3a84cd6b0c',
+  });
+}
 
-let fb = firebase.initializeApp(firebaseConfig);
-
-export { fb };
+const db = firebase.firestore();
```

Comment:
This needs to change to this based on the above suggestion.
```suggestion
const db = firebaseInstance.firestore();
```